config/deploy/production.rb

```diff
@@ -44,8 +44,8 @@
     rbenv_root: '/usr/local/rbenv'
 }
 set :rbenv_roles, :all
-set :rbenv_ruby, '2.0.0-p647'
-set :rbenv_ruby_dir, '/usr/local/rbenv/versions/2.0.0-p647'
+set :rbenv_ruby, '2.2.10'
```

Comment:
There's duplicated `:rbenv_ruby` in `config/deploy/*` files. Could you please remove one of the two?